src/test/resources/application.properties

```
@@ -1,2 +1,7 @@
 server.port=8888
-spring.datasource.url=jdbc:h2:mem:test;NON_KEYWORDS=USER
\ No newline at end of file
+#spring.datasource.url=jdbc:h2:mem:test;NON_KEYWORDS=USER
+spring.datasource.url=jdbc:postgresql://localhost:5434/jpa
+spring.datasource.username=postgres
+spring.datasource.password=admin
+# Hibernate ddl auto (create, create-drop, validate, update)
+spring.jpa.hibernate.ddl-auto=validate
```

```suggestion
spring.jpa.hibernate.ddl-auto=update
spring.liquibase.enabled=false
```
В тестах ликвибейс не нужен. Подправь эти проперти